mantidproject/mantid
# Adding ALF view

**Description of work.**
The direct group at ISIS have an alignment experiment called ALF. This PR is for a new GUI to determine if the sample is correctly aligned. This will replace the current software solution, which uses open genie. 



**To test:**
Open Mantid
Go to direct and select ALF view
The load bar will automatically search for ALF runs. Enter 80511
The instrument view will update.
The instrument view should not include the help button, the last two tabs (should only be pick and render). 
Go to the pick tab and select the `whole tube` option
Go to the plot on the instrument view and left click and select `integrate`
Left click the plot again and make sure the`axis` is set to `out of plane angle`.
Left click on a bright spot and select `export tube`
The plot on the right hand window should update
Click the `fit` button and a red fit should appear on the plot above. 
If you left click a tube again an extra option of `add to average` should appear. This will update the plot on the right hand side.
The `add to average` option will only appear if a tube has already been exported and will be removed when the run changes. The plot will not be cleared. 

You should be able to change runs and repeat the analysis.

Check instrument view has not changed. 
 


Fixes #26544 . 




---
In release notes
#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
